README.md

```diff
@@ -28,9 +28,9 @@ Follows these steps to build the docs locally:
 
 Install the dependencies:
 
-For Debian/Ubuntu: ``sudo apt-get install libpython-dev libssl-dev``
+For Debian/Ubuntu: ``sudo apt-get install libpython-dev libssl-dev libldap2-dev libsasl2-dev``
 
-For RHEL/CentOS: `` sudo yum install python-devel openssl-devel gcc``
+For RHEL/CentOS: `` sudo yum install python-devel openssl-devel gcc libldap2-dev libsasl2-dev``
```

Comment:
`libldap2-dev` and `libsasl2-dev` packages probably doesn't exist under the same name as in Ubuntu.
https://github.com/stackstorm/st2-auth-ldap#requirements however recommends another package for RHEL/CentOS

Reply:
are you referring to openldap-devel?

Reply:
Yes!